R/z_pages.R

```diff
@@ -86,18 +89,20 @@ animint2pages <- function(plot.list, github_repo, commit_message = "Commit from
     silent = TRUE
   )
   if (!has_commits) {
-    initial_commit(local_clone, repo, viz_url)
+    title <- plot.list[["title"]]
+    if(!is.character(title))title <- "New animint visualization"
+    initial_commit(local_clone, repo, viz_url, title)
   }
   # Handle gh-pages branch
   manage_gh_pages(repo, to_post, local_clone, commit_message)
   message(sprintf(
     "Visualization will be available at %s\nDeployment via GitHub Pages may take a few minutes...", viz_url))
-  viz_owner_repo
+  list(owner_repo=viz_owner_repo, local_clone=local_clone, viz_url=viz_url, gh_pages_url=sprintf("https://github.com/%s/tree/gh-pages", viz_owner_repo))
```

Comment:
Hi @tdhock , is the list for testing purpose?

Reply:
yes the list is for testing.
specifically local_clone is used to examine what files have been created (README.md title is tested)